server/routes/contact.js

```diff
@@ -9,7 +9,7 @@ router.get("/", (req, res) => {
 
 // /api/contact/messages
 router.get("/messages", (req, res) => {
-  Contact.find().then(contact => res.status(200).json(contact));
+  Contact.find().sort({ _id: -1 }).then(contact => res.status(200).json(contact));
```

Comment:
let's try to put this part in the front end, because we might need different sorts, it is not good to fetch the server every time I want the same data in a different order.